<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardioWorkoutDataUI</class>
 <widget class="QMainWindow" name="CardioWorkoutDataUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>384</width>
    <height>297</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter Pulse Rate Data</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioToday">
        <property name="text">
         <string>Today</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioDateSelect">
        <property name="text">
         <string>Specific Date</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDateEdit" name="dateSelect">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Run Time</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Run Distance</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Average Run Speed:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelRunSpeed">
        <property name="text">
         <string>1 mph</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="spinRunTime">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>24.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>hours</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QDoubleSpinBox" name="spinRunDistance">
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>24.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>miles</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>331</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>195</y>
       <width>169</width>
       <height>32</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSubmit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>384</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>radioDateSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSubmit</sender>
   <signal>clicked()</signal>
   <receiver>CardioWorkoutDataUI</receiver>
   <slot>submitData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CardioWorkoutDataUI</receiver>
   <slot>cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRunDistance</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CardioWorkoutDataUI</receiver>
   <slot>updateRunSpeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinRunTime</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CardioWorkoutDataUI</receiver>
   <slot>updateRunSpeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submitData()</slot>
  <slot>cancel()</slot>
  <slot>updateRunSpeed()</slot>
 </slots>
</ui>
